static/src/javascripts/projects/commercial/modules/header-bidding/a9/a9.ts

```diff
@@ -46,38 +50,36 @@ const requestBids = (advert, slotFlatMap) => {
 		return requestQueue;
 	}
 
-	const adUnits = getHeaderBiddingAdSlots(advert, slotFlatMap)
-		.map((slot) => new A9AdUnit(advert, slot))
-		.filter((adUnit) => !adUnit.isEmpty());
```

Comment:
I think this is redundant as our types are enforcing that `slotID` is non-null.

Reply:
Fair enough